resources/README.txt

```diff
@@ -0,0 +1,12 @@
+The "external.yaml" file here is produced by running:
```

Comment:
I don't see an external.yaml in this pull request.

Reply:
Jenkins just pointed out the same thing.  `.gitignore` was hiding the fact that I hadn't committed it.